scripts/generate_account.py

```diff
@@ -13,17 +14,20 @@
     metavar="KEY",
     help="Specify the private key to use instead of generating one.",
 )
+@click.option(
+    "--password", type=str, required=True, help="Choose a password to encrypt the keystore file"
```

Comment:
Without additional changes, making this `required` will break our E2E tests. I would actually prefer if this option is left... optional and instead make the default password empty.

Reply:
IMHO, if it's only about our own e2e test, I'm more than happy to change it to make it work. I would really not give this script into the hands of the user, with making default passwords. 

So we should either don't promote the option to the user at all and then we can use it as an internal script which is not meant for external usage. (The contributing page should be updated, because it implicitly sets the empty password and the command for running an agent).

But as one of our real agents pointed rightfully out, they asked for a tool to generate the key locally reducing the risk of external exposure by not needing to send keys through the network.
If we promote this option, I am really not a fan of setting implicit passwords, because the user might not be aware and this could be a potential agent key with funds on it.

Additionally, I think it's not a big deal to add `--password ""` to our e2e commands.

Reply:
> 



> IMHO, if it's only about our own e2e test, I'm more than happy to change it to make it work. I would really not give this script into the hands of the user, with making default passwords.
>
> So we should either don't promote the option to the user at all and then we can use it as an internal script which is not meant for external usage. (The contributing page should be updated, because it implicitly sets the empty password and the command for running an agent).

Yeah, I never really imagined the script to be used in this manner, but OTOH I see no issues. I somehow hoped
whoever is going to run an agent is going to be knowledgeable enough and also have other (more trusted) ways to generate an address, or already have an address ready. But I also don't have anything against them using the script.

> But as one of our real agents pointed rightfully out, they asked for a tool to generate the key locally reducing the risk of external exposure by not needing to send keys through the network. If we promote this option, I am really not a fan of setting implicit passwords, because the user might not be aware and this could be a potential agent key with funds on it.
> 
> Additionally, I think it's not a big deal to add `--password ""` to our e2e commands.

Sure, please go for it. :+1: